posydon/binary_evol/SN/step_SN.py

```diff
@@ -70,6 +70,7 @@
     "engine": 'N20',
     "PISN": "Marchant+19",
     "ECSN": "Podsiadlowksi+04",
+    "conserve_hydrogen_envelope" : False,
```

Comment:
Just a small naming thing: I wouldn't call it `conserve_hydrogen_envelope`, because we don't conserve the hydrogen-rich envelope. The NS/BH won't have a hydrogen-rich envelope. We include/consider it in the collapse. Thus, I'd suggest names like `include_hydrogen_envelope` or `collapse_hydrogen_envelope`.